components/autofill/src/sync/mod.rs

```diff
@@ -18,6 +18,10 @@ use sync15::{CollectionName, ServerTimestamp};
 use sync_guid::Guid;
 use types::Timestamp;
 
+// This type is used as a snazzy way to capture all unknown fields from the payload
+// upon deserialization without having to work with a concrete type
+pub(super) type UnknownFields = serde_json::Map<String, serde_json::Value>;
```

Comment:
I think `super` here is the crate. `pub(self)` is probably all you need here as nothing outside the sync module knows about this. But, in that case and because this is in the root of the crate, I'm not sure you need *any* pub here?